examples/annotation/detect_hijacks.py

```diff
@@ -25,58 +32,105 @@ def choose_input(input):
                         help="collector name from where the log files are",
                         default="none")
     parser.add_argument("-i", "--input",
                         help="MRT parser, e.g. 'mabo'",
                         default="mabo")
-    parser.add_argument("-o", "--options",
-                        help="extra options passed to the input method")
-    parser.add_argument("--irr-ro-file",
-                        help="CSV file containing IRR route objects")
-    parser.add_argument("--irr-mnt-file",
-                        help="CSV file containing IRR maintainer objects")
-    parser.add_argument("--irr-org-file",
-                        help="CSV file containing IRR organisation objects")
-    parser.add_argument("--rpki-roa-file",
-                        help="CSV file containing ROA")
+    
+    parser.add_argument("--registry-path",
+                        help="enter the path for registry data")
+
+    parser.add_argument("--bgp-path",
+                        help="enter the path for bgp rib and update files")
+    
     parser.add_argument("-v", "--verbose", action="store_true",
                         help="more logging")
 
-    args = parser.parse_args()
+    args = parser.parse_args()    
 
     logging.basicConfig(level=logging.DEBUG if args.verbose else logging.INFO)
-
-    input_kwargs = {"files": args.mrt_files}
-    if args.options is not None:
-        try:
-            input_kwargs.update(dict([w.split("=", 2) for w in
-                                      args.options.split(",")]))
-        except:
-            raise ValueError("invalid input options (use format key=value)")
-
-    input = choose_input(args.input)
-    kwargs = input(args.collector, **input_kwargs)
-
-    if args.irr_ro_file is not None:
-        kwargs["irr_ro_file"] = args.irr_ro_file
-
-    if args.rpki_roa_file is not None:
-        kwargs["rpki_roa_file"] = args.rpki_roa_file
-
-    if args.irr_org_file is not None:
-        kwargs["irr_org_file"] = args.irr_org_file
-
-    if args.irr_mnt_file is not None:
-        kwargs["irr_mnt_file"] = args.irr_mnt_file
-
-    # detect the conflicts and print them
-    for conflict in detect_hijacks(**kwargs):
-        if conflict["type"] == "ABNORMAL":
-            print(json.dumps(conflict))
+    logging.basicConfig(level=logging.INFO,
+                        format='%(asctime)s - %(message)s',
+                        datefmt='%Y-%m-%d %H:%M:%S')
+    
+    kwargs = {}
+    mrt_files = []    
+    targets = []
+    targets.append(args.registry_path)
+    targets.append(args.bgp_path)
+    event_handler = PollingHandler()
+    observer = Observer()
+    observers = []
+
+    for path in targets:
+        targetPath = str(path)
+        observer.schedule(event_handler, targetPath, recursive=False)
+        observers.append(observer)
+    
+    observer.start()
+
+    try:
+        while True:            
+            time.sleep(1)
+    except KeyboardInterrupt:
```

Comment:
If we are running this in background, will there be keyboard interrupt? I don't understand this part well. Check it out again.